pkg/erofs/verity.go

```diff
@@ -0,0 +1,5 @@
+package erofs
+
```

Comment:
Could use a comment here, similar to the squashfs which describes how squashfs pads/aligns its data.

Reply:
Done.